### PR TITLE
New entry: HTML `<noscript>` tag

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added an entry for the `<noscript>` tag.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above)

### CODE REVIEW

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:3]:
```suggestion
Description: 'Displays content within if Javascript is disabled in the browser or not supported.'
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:14]:
```suggestion
The `<noscript>` tag displays its contents whenever [Javascript](https://www.codecademy.com/resources/docs/javascript) is disabled on the browser or it is not supported for a given site.
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:20]:
```suggestion
  <!-- What should display if JavaScript is not enabled or supported. -->
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:24]:
```suggestion
The tag has no attributes and what is nested within the tag will be displayed.
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:30]:
```suggestion
<noscript>
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:35]:
```suggestion
The output might look like this:
```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:29]: What is the purpose of this `<script>` tag being here?

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding Finished with my review and left you some comments/suggestions

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (ethan-master-coding) [reply content/html/concepts/tags/terms/noscript/noscript.md:29]: @Dusch4593 Why would you use the no script tag if you don’t have JavaScript?

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes.

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [reply content/html/concepts/tags/terms/noscript/noscript.md:29]: I'm saying that if the `<script>` tag plays a role in this example, it should be explained in text-form above the snippet (but below the "Example" heading).

Also, `href` is not a valid attribute with the `<script>` tag; use `src="someScript.js"`, instead.

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:36]:
```suggestion

The output might look like this:

```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:17]:
```suggestion

```

File: content/html/concepts/tags/terms/noscript/noscript.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:15]:
```suggestion

```

**Review by Dusch4593 — APPROVED** [review]
@ethan-master-coding LGTM! 👍🏻 Let's have one more reviewer look at this before we merge.

**Review by SSwiniarski — APPROVED** [review]
@ethan-master-coding, @Dusch4593: Looks good. I don't have any more suggestions.

**Comment by Dusch4593** [comment]
@ethan-master-coding @SSwiniarski Merging. Also, I forgot that we still needed a sentence or two right below the "Example" heading, so I added one in cd82f093a84f9310b2f39e14acd57bc232d17a90.

**Comment by ethan-master-coding** [comment]
@Dusch4593 we have 2 sentences saying simpler things for the example (lines 28 and 40). Should I open a PR to remove the second one?

**Comment by Dusch4593** [comment]
> @Dusch4593 we have 2 sentences saying simpler things for the example (lines 28 and 40). Should I open a PR to remove the second one?

@ethan-master-coding I see what you're saying. Instead of removing the second sentence, let's open a PR that simply removes the latter part about JavaScript being disabled or not supported.

**Comment by ethan-master-coding** [comment]
For feature visitors #783

**Pull request merged**
